src/store/useChannelStore.js

```diff
@@ -5,12 +5,14 @@ export const useChannelStore = create(
   persist(
     (set) => ({
       interestChannelIds: [],
+      isChannelChanged: false,
       prevChannelId: null,
       radioChannelList: [],
       selectedChannelId: null,
 
       setInterestChannelIds: (interestChannelIds) =>
         set({ interestChannelIds }),
+      setIsChannelChanged: (isChanged) => set({ isChannelChanged: isChanged }),
```

Comment:
```suggestion
      setIsChannelChanged: (isChannelChanged) => set({ isChannelChanged }),
```
위 `setInterestChannelIds`처럼 `setIsChannelChanged`도 이렇게 리팩토링 할 수 있을 것 같아요! 둘 중 어떤 방식이 더 마음에 드시나요? @faber-h @Eun0713

Reply:
두 방식 다 괜찮은 것 같아요! 그렇지만 개인적으로는 재윤님이 제안주신 방식이 조금 더 깔끔해보이는 것 같습니다☺️

Reply:
좋은 포인트 감사합니다!
말씀 주신 방식이 가독성 측면에서 더 좋은 것 같아요! 😀

다만, 지금 코드베이스를 보니 전반적으로 혼재되어 있어서,
이 부분은 나중에 따로 이슈로 파서 전반적인 리팩토링을 진행하도록 하겠습니다!

Reply:
[Zustand 상태 업데이트 네이밍 및 함수 작성 스타일 통일](https://github.com/Radio-Premium/RadioPremium-FE/issues/162) 해당 issue에서 `set({ key: value })` 형태를 `set({ key })`로 통일하였습니다!